frontend/src/components/ControleLeitos/HeaderControleLeitos.vue

```diff
@@ -0,0 +1,31 @@
+<template>
+  <v-container fluid>
+    <v-row justify="space-between" align="center">
+      <v-col cols="auto">
+        <h4 class="primary--text my-7 m-0 mb-0 display-1">Controle de Leitos</h4>
+      </v-col>
+      <v-col cols="auto">
+        <v-btn
+          v-show="isSecretariaSaude"
```

Comment:
Teria q confirmar, acho q quem pode adicionar os leitos é ou a secretaria ou se o leito for da unidade de saúde do usuário logado.